src/GraphQL/Validation/TypeInfo.cs

```diff
@@ -18,7 +17,6 @@ public class TypeInfo : INodeVisitor
         private readonly Stack<INode> _ancestorStack = new Stack<INode>();
         private DirectiveGraphType _directive;
         private QueryArgument _argument;
-
```

Comment:
to no purpose |  
for nothing

Reply:
not sure why. i'll put the empty line back in there.

Reply:
probably a merge conflict, actually.

Reply:
fixed